### ISSUE TITLE
mammary gland tumor multiplicity 

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/mammalian-phenotype-ontology#3166]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. Abnormal tumor multiplicity
2. Increased mammary gland tumor multiplicity
3. Decreased mammary gland tumor multiplicity 

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. deviation from the expected number of tumors per individual subject
2. Increased mean number of mammary gland tumors per individual subject.
3. Decreased mean number of mammary gland tumors per individual subject.

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

Consider "incidence (number of affected mice/total number of mice available for examination)", and "Tumor multiplicity (either total or malignant tumors only) is defined as the number of tumors per mouse either of a particular organ or inclusively at any site". PMID:14514661

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. "altered tumor susceptibility" (MP:0002166)
2 & 3. "abnormal tumor multiplicity" (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:cp
